### ISSUE TITLE
[Source Request]: Silea - Italy

### ISSUE THREAD

**Issue by mvimercati** [mampfes/hacs_waste_collection_schedule#2953]
### Municipality / Region

Italy

### Collection Calendar Webpage

https://www.sileaspa.it/calendario-raccolta-rifiuti/

### Example Address

In the form: 
City: Lomagna
Address: via Martiri della libertà
Month: Ottobre

### Collection Data Format

Something different (add to additional information)

### Additional Information

I can get a calendar for the remaining days in a month (id_mese -> 10 october) by issuing this POST request:
`
curl -d "action=get_calendar&id_cliente=1056&id_via=22225&id_mese=10" "https://www.sileaspa.it/wp-admin/admin-ajax.php"
`

The response is a simple json:
```
{
  "30 mer": {
    "date_formatted": "30 mer",
    "date": "2024-10-30T00:00:00",
    "services": [
      {
        "service": "Spazzamento meccanizzato",
        "service_original": "Spazzamento meccanizzato ",
        "hours": "09:00 - 18:00",
        "type": "Pulizia Meccanizzata"
      }
    ]
  },
  "31 gio": {
    "date_formatted": "31 gio",
    "date": "2024-10-31T00:00:00",
    "services": [
      {
        "service": "Raccolta vetro",
        "service_original": "Raccolta vetro",
        "hours": "09:00 - 18:00",
        "type": "raccolta VETRO"
      },
      {
        "service": "Raccolta carta",
        "service_original": "Raccolta carta ",
        "hours": "09:00 - 18:00",
        "type": "raccolta CARTA CARTONE"
      }
    ]
  }
}
```

**Comment by mvimercati** [comment]
Implemented in  #3061

**Comment by mvimercati** [comment]
Merged in master https://github.com/mampfes/hacs_waste_collection_schedule/commit/e9bfb6f18ed082f735f0676d0bebde6ca8807353
